integration-tests/jackson/Readme.md

```diff
@@ -0,0 +1,109 @@
+# Quarkus Jackson extension tester
+
+This project verifies Jackson (de-)serialization support in native mode. 
+
+This project consists of the following modules:
+
+- model
+
+    This module contains the library with a simple models. This model makes use 
+    of Jackson to support (de-)serialization to JSON. Various forms of models 
+    exists. At the time of writing the following models are present/tested:
+    - Immutable models using a Builder to construct new instances.
+    - Simple POJO model being registered for reflection.
+    
+    Unittests are available to prove JVM based JSON (de-)serialization works 
+    properly.
+
+- service 
+
+    This module contains a very simple restful resources with only a post method
+    to post new models to this service. A simple unittest is included to verify
+    correct working for both unit and integration test.
```

Comment:
`correct working` should probably be `correct behavior`

Reply:
You used the British English spelling :). Actually I don't think we have ever addressed this (obviously minor) issue in the guides.
So let's keep it the way you did it :)